web-components/src/components/credits/CreditDetails.tsx

```diff
@@ -180,34 +119,38 @@ export default function CreditInfo({
         </div>
         {creditClass.description && (
           <div className={classes.descriptionItem}>
-            <Typography component="div" className={classes.description}>
+            <Body
+              size="xl"
+              sx={{ mt: 3.75, mr: { sm: background ? 11.75 : 0 } }}
+            >
               {ReactHtmlParser(creditClass.description)}
-            </Typography>
+            </Body>
           </div>
         )}
         <div className={classes.activitiesTitleContainer}>
-          <Title className={classes.activitiesTitle} variant="h4">
+          <Title variant="h4" mobileVariant="h6" sx={{ mt: [null, 3.5] }}>
```

Comment:
the null value in `sx={{ mt: [null, 3.5] }}` is saying skip that breakpoint. The same as `[theme.breakpoints.up('sm')` in this case (or `sx={{ mt: { sm: 3.5 } }}`